sdcm/mgmt/cli.py

```diff
@@ -213,20 +213,24 @@ def arguments(self) -> str:
                 break
         return arguments_string
 
+    def progress_string(self, **kwargs):
```

Comment:
please add a docstring explaining what this funcion is doing and what will be returned.